charts/rancher-vsphere-cpi/rancher-vsphere-cpi/100.0.0/values.yaml

```diff
@@ -14,6 +14,11 @@ cloudControllerManager:
     repository: rancher/mirrored-cloud-provider-vsphere-cpi-release-manager
     tag: v1.2.1
   nodeSelector: {}
+  rbac:
+    enabled: true
```

Comment:
What if the user sets it to be false? 
Will the app fail to install because of the missing service account?

Reply:
No, it doesn't fail installation. This is just to provide the option to disable the chart's RBAC to avoid incompatibilities like those stated in the issue for this PR (conflicts with rke2's RBAC), but then the expectation is the user will set it up themselves. This is not in the chart's questions so it's something they would need to set up manually.